packages/sdk/src/credential/onBehalfOfUserCredential.ts

```diff
@@ -214,4 +231,37 @@ export class OnBehalfOfUserCredential implements TokenCredential {
       return new ErrorWithCode(fullErrorMsg, ErrorCode.ServiceError);
     }
   }
+
+  private parseCertificate(certificatePath: string | undefined): ClientCertificate | undefined {
+    if (!certificatePath) {
+      return undefined;
+    }
+
+    const certificateContent = fs.readFileSync(certificatePath, "utf8");
+    const certificatePattern =
```

Comment:
is there any existing method to load certificate?

Reply:
I just refer to code from azure/identity lib: https://github.com/Azure/azure-sdk-for-js/blob/02bcfb182851c6b819dff4ac215318d8b27244e1/sdk/identity/identity/src/msal/nodeFlows/msalClientCertificate.ts#L58